Source/Core/DolphinQt/Updater.h

```diff
@@ -15,13 +15,13 @@ class Updater : public QThread, public AutoUpdateChecker
 {
   Q_OBJECT
 public:
-  explicit Updater(QWidget* parent);
+  explicit Updater(QWidget* parent, bool silently_fail);
```

Comment:
Please use an enum class rather than a boolean to improve readability at call sites. Something like `enum class ShouldSilentlyFail { Yes, No }` would be a lot clearer than seeing a random false or a true.

Reply:
Does this even have to be `explicit` when it has more than one parameter?